config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -423,7 +423,7 @@ presubmits:
         - --provider=gce
         - --runtime-config=api/all=true
         - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gce-alpha-features
-        - --test_args=--ginkgo.focus=\[Feature:(GRPCContainerProbe|InPlacePodVerticalScaling|ProbeTerminationGracePeriod|APIServerTracing|StorageVersionAPI|PodPreset|StatefulSetMinReadySeconds|CustomResourceValidationExpressions)\]|Networking --ginkgo.skip=\[Feature:(SCTPConnectivity|Volumes|Networking-Performance)\]|IPv6|csi-hostpath-v0 --minStartupPods=8
+        - --test_args=--ginkgo.focus=\[Feature:(GRPCContainerProbe|InPlacePodVerticalScaling|ProbeTerminationGracePeriod|APIServerTracing|StorageVersionAPI|PodPreset|StatefulSetMinReadySeconds|CustomResourceValidationExpressions|AnyVolumeDataSource)\]|Networking --ginkgo.skip=\[Feature:(SCTPConnectivity|Volumes|Networking-Performance)\]|IPv6|csi-hostpath-v0 --minStartupPods=8
```

Comment:
Is there a Kubernetes PR validating that this test passes?

Reply:
Ah, I see https://github.com/kubernetes/kubernetes/pull/108664, but do you know if this test ran? I think there's a way to configure a Kubernetes PR to run with a prow config from a PR, but I can't recall the details.

Reply:
@andrewsykim Thank you for your review. IIUC, without `--ginkgo.focus` option for `"[Feature:AnyVolumeDataSource]"`, the e2e tests containing `"[Feature:AnyVolumeDataSource]"` string are  skipped. So, this PR is needed to make this job not to skip these tests. In [the PR](https://github.com/kubernetes/kubernetes/pull/108664), the job is enabled with `/test pull-kubernetes-e2e-gce-alpha-features`, but the tests seem to be [skipped](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/108664/pull-kubernetes-e2e-gce-alpha-features/1505970126862684160/).

Reply:
This is kind of hacky, but could you temporary update the ginkgo tag in your PR to something else to have the test run and see if it passes?

Reply:
The last commit which changed [this line](https://github.com/kubernetes/test-infra/blame/master/config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml#L426) was https://github.com/kubernetes/test-infra/pull/24393, and it enables `[Feature:CustomResourceValidationExpressions]` for the [tests]( https://github.com/kubernetes/kubernetes/pull/106462/files#diff-ac9cb49702125d3dc287a64f6f2b98121d31ede705d69eed94aa875172149824R37) in k/k by this way. So, I think that this is the way the tests for alpha features have been enabled (and I agree that it looks hacky).

Reply:
I'm not disagreeing with the approach to enable the tests, just asking if we can have a successful run of the test before enabling this for all alpha jobs. The (hacky) approach I mentioned is just a temporary hack to run the test in presubmit by using a known filter that works (just pick any of the existing alpha filters that are used).

Reply:
The test's PR is tested manually like [this](https://github.com/kubernetes/kubernetes/pull/108664#issuecomment-1067365541). The test requires alpha features, or AnyVolumeDataSource feature gate, to be enabled, which means removing `[Feature:AnyVolumeDataSource]` will make this test run in other jobs which don't enable the feature gate and fail. 

So, I will update the test's PR if it is really required before merging this PR, however, I don't think that it should be a blocker for this PR (This PR could be merged without any side effects and the test's PR can be merged later without any dependencies).

Reply:
> The test's PR is tested manually like https://github.com/kubernetes/kubernetes/pull/108664#issuecomment-1067365541. The test requires alpha features, or AnyVolumeDataSource feature gate, to be enabled, which means removing [Feature:AnyVolumeDataSource] will make this test run in other jobs which don't enable the feature gate and fail.

This seems sufficient, thanks